cluster-autoscaler/utils/drain/drain.go

```diff
@@ -38,6 +39,8 @@ const (
 	// PodSafeToEvictKey - annotation that ignores constraints to evict a pod like not being replicated, being on
 	// kube-system namespace or having a local storage.
 	PodSafeToEvictKey = "cluster-autoscaler.kubernetes.io/safe-to-evict"
+	// SafeToEvictLocalVolumeKey - annotation that ignores (doesn't block on) a local storage volume during node scale down
+	SafeToEvictLocalVolumeKey = "cluster-autoscaler.kubernetes.io/safe-to-evict-local-volume"
```

Comment:
I have changed the annotation key here to `safe-to-evict-local-volume` instead of the suggested `local-volume-safe-to-evict` because I think it's easier to read and remember. This is of course my thinking. Happy to change it based on review comments.

Reply:
Annotation key name is being discussed in this comment: https://github.com/kubernetes/autoscaler/pull/5594#issuecomment-1486257828

Reply:
Makes sense :+1:

Reply:
Updated to `safe-to-evict-local-volumes`